lib/bigcommerce.js

```diff
@@ -191,6 +191,14 @@ class BigCommerce {
     });
   }
 
+  addParamsToPath(path, params) {
+    if (!params || Object.keys(params).length === 0  ) {
+      return path;
+    }
+    const paramsString = new URLSearchParams(params);
+    return `${path}?${paramsString.toString()}`;
```

Comment:
I should probably make this combine with any existing params already on the `path`, folks will inevitably put them there also.